### PR TITLE
Improve serialization of Height and add BigInt serialization

### PR DESCRIPTION
**Pull request by Geod24**
This is a prelude to completely switching over to BigInt.

### CODE REVIEW

File: src/Stoa.ts [hunk 1]
```diff
@@ -15,6 +15,13 @@ import express from 'express';
 import { UInt64 } from 'spu-integer-math';
 import { URL } from 'url';
 
+// Module extension to allow customizing JSON serialization
+declare global {
+    interface BigInt {
+        toJSON(key? : string): string;
```
Comment (TrustHenry) [inline src/Stoa.ts]:
```suggestion
        toJSON(key?: string): string;
```

File: src/Stoa.ts [hunk 2]
```diff
@@ -69,6 +76,11 @@ class Stoa extends WebService
         {
             return this.task();
         }, 10);
+
+        // Allow JSON serialization of BigInt
+        BigInt.prototype.toJSON = function(key? : string) {
```
Comment (TrustHenry) [inline src/Stoa.ts]:
```suggestion
        BigInt.prototype.toJSON = function(key?: string) {
```

File: src/modules/data/Height.ts [hunk 3]
```diff
@@ -64,4 +64,15 @@ export class Height
     {
         return Utils.UInt64ToString(this.value);
     }
+
+    /**
+     * Converts this object to its JSON representation
```
Comment (TrustHenry) [inline src/modules/data/Height.ts:69]:
```suggestion
     * Converts this object to it's JSON representation
```

File: src/modules/data/Height.ts [hunk 4]
```diff
@@ -64,4 +64,15 @@ export class Height
...
+     *
+     * Use `string` as primitive types, as JS is only precise up to
+     * `2 ** 53 - 1` but we can get numbers up to `2 ** 64 - 1`.
+     */
+    public toJSON (key?: string) : string
```
Comment (TrustHenry) [inline src/modules/data/Height.ts]:
```suggestion
    public toJSON (key?: string): string
```

**Review by TrustHenry — APPROVED** [review]
Looks good, just some styling nits.

File: src/modules/data/Height.ts (hunk 3 above)
Comment (Geod24) [reply src/modules/data/Height.ts:69]: In this case, `its` is correct.
https://www.grammarly.com/blog/its-vs-its
And yes, it's (!) weird to me too ;)

Comment (TrustHenry) [reply src/modules/data/Height.ts:69]: Oh, TIL

**Comment by Geod24** [comment]
Comments addressed

**Pull request merged**